src/components/ValuesScreen.tsx

```diff
@@ -1,17 +1,426 @@
-import React from 'react';
-import { View } from 'react-native';
-import { Text } from 'react-native-paper';
-
+import React, {useEffect, useState} from 'react';
+import { View, ScrollView, Dimensions, TouchableWithoutFeedback, Keyboard } from 'react-native';
+import { Text, Title, Button, TextInput, FAB, IconButton, Surface, useTheme, Portal, Dialog, Paragraph} from 'react-native-paper';
 import { createStackNavigator } from '@react-navigation/stack';
 import { CustomNavigationBar } from './CustomNavigationBar';
+import { useTranslation } from 'language/LanguageProvider';
+import Storage from 'storage';
+import { ValuesTopic, People, ValuesEntry } from 'storage/types';
+import { IconMeny } from './IconMeny';
+import { IconList } from './activity/IconList';
+import { useLinkProps } from '@react-navigation/native';
 import { SettingsScreen } from './SettingsScreen';
 
 const ValuesStack = createStackNavigator();
 
-const ViewContent = () => {
+const RoundButton = (props: any) => {
+  return (
+    <Surface style={{ borderRadius: 100, elevation: 3}}>
+      <IconButton icon={props.icon} size={props.size} onPress={props.onPress} />
+    </Surface >
+  );
+  }
+  
+const CircleButton = (props: any) => {
```

Comment:
```suggestion
const CircleButton = (props: {icon: string, size: number,  onPress: () => void}) => {
```
Props can be typed by doing something similar.
This should be done for all cases when applicable.